### PR TITLE
Add search tip

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add tip for people who struggle finding specific hooks that are described in a generic way

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ A couple of hooks were modified between 1.7.0.x and 1.7.1.x.
 
 ## Full list
 
+{{% notice tip %}}
+**Tip:** Some hooks whose names include a page or component name are described generically using placeholders. If you can't find a hook like `actionAdminCustomersFormModifier`, try searching for `FormModifier`. 
+{{% /notice %}}
+
```
Comment (PululuK) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md:25]: maybe we should clarify that these are the dynamic hooks, which is why we can list them all, which is why we have written them in a generic way.

Comment (eternoendless) [reply src/content/1.7/modules/concepts/hooks/list-of-hooks.md:25]:
```suggestion
{{% notice tip %}}
**Search tip:** Some hooks are generated dynamically, so their names are documented in a generic way.

For example, `actionAdminCustomersFormModifier` is documented as `action<AdminControllerClassName>FormModifier`, so you won't find it if you search for the exact name. When you see a controller name or action in the hook name and you can't find it, try searching for a part of the hook name, like `FormModifier`.
{{% /notice %}}

```

Comment (PululuK) [reply src/content/1.7/modules/concepts/hooks/list-of-hooks.md:25]: Yeah ✌🏿

**Comment by eternoendless** [comment]
Thanks for the review!

**Pull request merged**